docs/modules/ROOT/pages/guides/weights_fees.adoc

```diff
@@ -0,0 +1,40 @@
+:source-highlighter: highlight.js
+:highlightjs-languages: rust
+:github-icon: pass:[<svg class="icon"><use href="#github-icon"/></svg>]
+
+= Weights & Fees
+
+Weight is a metric to estimate the time it takes to execute code.
+
+In Substrate, every transaction has a weight. The block production algorithm selects the set of transactions from the transaction pool that achieve block saturation without exceeding `MaximumBlockWeight`. The `AvailableBlockRatio` ensures only a fraction of `MaximumBlockWeight` is used for regular transactions while system-critical operations (operational transactions) may use all remaining block weight.
```

Comment:
> The AvailableBlockRatio ensures only a fraction of MaximumBlockWeight is used for regular transactions while system-critical operations (operational transactions) may use all remaining block weight.

I have read this 4 times, and it is still not clear to me. It could be that it's late for me :)